app/commands/v2/put_content.rb

```diff
@@ -45,6 +47,26 @@ def create_or_update_draft_content_item!
       def content_item_attributes
         payload.slice(*DraftContentItem::TOP_LEVEL_FIELDS)
       end
+
+      def handle_path_change(content_item)
+        path_change = content_item.previous_changes[:base_path]
```

Comment:
I'd have been tempted to use destructuring assignment:
`old_path, new_path = content_item.previous_changes[:base_path]`